compiler/rustc_middle/src/ty/layout.rs

```diff
@@ -1101,12 +1101,11 @@ where
 ///
 /// This takes two primary parameters:
```

Comment:
this is now also wrong? :grinning: should this mention `fn_can_unwind` as that was probably used to replace `codegen_fn_attr_flags`?

Reply:
Sorry  I am not sure which line you are talking about.

Reply:
woops, meant `fn_def_id`.

> /// This takes two primary parameters:

Either say "this takes one primary parameter"/remove this line entirely, or alternatively, also mention `fn_def_id` and mention "This is only applicable for Rust-defined functions, and generally isn't needed except for small optimizations where we try to say a function which otherwise might look like it could unwind doesn't actually unwind (such as for intrinsics and such)."

Reply:
I did already mentioned fn_def_id though?

Reply:
I might have still been a bit too tired? :sweat_smile: yikes

sorry, you're absolutely right